src/draive/conversation/completion/state.py

```
@@ -54,44 +53,52 @@ async def completion(
                 stream=True,
             )
 
         else:
-            return await conversation.completing(
+            return await ctx.state(cls).completing(
                 instruction=Instruction.of(instruction),
-                input=input,
+                input=conversation_message,
                 memory=conversation_memory,
                 toolbox=Toolbox.of(tools),
             )
```

Pylint (3.3.7)

[refactor] Unnecessary "else" after "return", remove the "else" and de-indent the code inside it

(R1705)